tools/src/main/java/io/github/ygojson/tools/dataprovider/impl/yugipedia/mapper/YugipediaPrintMapper.java

```diff
@@ -0,0 +1,129 @@
+package io.github.ygojson.tools.dataprovider.impl.yugipedia.mapper;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Stream;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.ReportingPolicy;
+
+import io.github.ygojson.model.data.Print;
+import io.github.ygojson.model.data.definition.localization.Language;
+import io.github.ygojson.tools.dataprovider.impl.yugipedia.model.CardTable2;
+import io.github.ygojson.tools.dataprovider.impl.yugipedia.model.MarkupString;
+
+@Mapper(
+	uses = { GeneralMapper.class },
+	unmappedTargetPolicy = ReportingPolicy.ERROR
+)
+public abstract class YugipediaPrintMapper {
+
+	private static final int EXPECTED_FIELD_NUMBER = 3;
+
+	// 1st field: cardNumber - out printCode
+	private static final int CARD_NUMBER_FIELD = 0;
+	// 2nd field: setName (useful for the firstSeries cards withut printCode)
+	private static final int SET_NAME_FIELD = 1;
+	// 3rd filed: rarity list
+	private static final int RARITY_FIELD = 2;
```

Comment:
Change name to `RARITY_LIST_FIELD` to be sure that we remember that is not a single rarity.